integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -523,9 +506,9 @@ def test_hash_multiple_grpby_pivot(data_gen, conf):
 @approximate_float
 @ignore_order(local=True)
 @incompat
-@pytest.mark.parametrize('data_gen', _init_list_no_nans, ids=idfn)
+@pytest.mark.parametrize('data_gen', _init_list, ids=idfn)
 @pytest.mark.parametrize('conf', get_params(_confs, params_markers_for_confs), ids=idfn)
-def test_hash_reduction_pivot_without_nans(data_gen, conf):
+def test_hash_reduction_pivot_with_nans(data_gen, conf):
```

Comment:
There's a test after this with the same name that clobbers this test (if it doesn't make pytest fail outright?)